cartographer/mapping_3d/local_trajectory_builder.cc

```diff
@@ -166,10 +166,13 @@ LocalTrajectoryBuilder::AddAccumulatedRangeData(
       filtered_range_data_in_tracking, pose_estimate.cast<float>());
   last_pose_estimate_ = {time, pose_estimate,
                          filtered_range_data_in_local.returns};
-  return InsertIntoSubmap(
+  std::unique_ptr<InsertionResult> insertion_result = InsertIntoSubmap(
       time, filtered_range_data_in_local, filtered_range_data_in_tracking,
       high_resolution_point_cloud_in_tracking,
       low_resolution_point_cloud_in_tracking, pose_estimate, gravity_alignment);
+  return std::unique_ptr<MatchingResult>(new MatchingResult{
```

Comment:
Do not use `new`. Prefer `common::make_unique<>()`. Similar below for the `InsertionResult`.

Reply:
Sorry, a new slipped here by accident. Where else did you see a `new` in this PR?